install.ps1

```diff
@@ -13,18 +13,20 @@
     .PARAMETER Version
     If provided, tries to download the specific version instead of the latest
     .EXAMPLE
-    Import-Module ./install.ps1; Install-Mondoo -RegistrationToken INSERTKEYHERE
+    Import-Module ./install.ps1; Install-Mondoo -RegistrationToken 'INSERTKEYHERE'
     Import-Module ./install.ps1; Install-Mondoo -Version 6.14.0
     Import-Module ./install.ps1; Install-Mondoo -Proxy 1.1.1.1:3128
     Import-Module ./install.ps1; Install-Mondoo -Service enable
     Import-Module ./install.ps1; Install-Mondoo -UpdateTask enable -Time 12:00 -Interval 3
     Import-Module ./install.ps1; Install-Mondoo -Product cnspec
+    Import-Module ./install.ps1; Install-Mondoo -Path 'C:\Program Files\Mondoo\'
```

Comment:
very cool, that also makes sure that cnquery / cnspec are downloaded to `C:\Program Files\Mondoo`